src/qt/addressbookpage.cpp

```diff
@@ -107,7 +107,7 @@ AddressBookPage::AddressBookPage(const PlatformStyle *platformStyle, Mode _mode,
         ui->newAddress->setVisible(true);
         break;
     case ReceivingTab:
-        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. Use the request payment button in the receive tab to create new addresses."));
+        ui->labelExplanation->setText(tr("These are your Bitcoin addresses for receiving payments. Use the 'Create new receiving address' button in the receive tab to create new addresses."));
```

Comment:
Nit: Prefer to have the button label here as a `%1` token, so translations don't need to change when/if we rename the button again. (OTOH, the button itself needs `&amp;`, so maybe not worth the trouble...)